docs/src/components/Search/primitives/StyledMenu.ts

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+
+export const StyledMenu = styled.ul<{ hasResults?: boolean }>`
+  ${({ hasResults = false }) => `
```

Comment:
why it's set by default as false? it's false even without default parameter

Reply:
Just a coding pattern I was trying out, as by default it's not `false`, it's `undefined`, but it's not important.

Reply:
yes, by false i meant falsy value, not false itself.